tests/e2e/local/vagrant/revert_dockerdaemon_linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Why do you need the Mac OS docker daemon png file?

Reply:
To show users where to add the insecure registry option more clearly.